wpimath/src/main/java/edu/wpi/first/math/geometry/Rotation2d.java

```diff
@@ -153,6 +153,25 @@ public Rotation2d rotateBy(Rotation2d other) {
         m_cos * other.m_cos - m_sin * other.m_sin, m_cos * other.m_sin + m_sin * other.m_cos);
   }
 
+  /**
+   * Returns the absolute value of the Rotation2d.
+   *
+   * @return The new absolute value Rotation2d
+   */
+  public Rotation2d abs() {
+    return new Rotation2d(Math.abs(m_value));
+  }
+
+  /**
+   * Inverts the Rotation2d, unlike `unaryMinus`, which is a flip across the x-axis,
+   * this rotates the angle 180 degrees about the origin.
+   *
+   * @return The new inverted Rotation2d
+   */
+  public Rotation2d invert() {
+    return rotateBy(new Rotation2d(Math.PI));
+  }
```

Comment:
This is incorrect for angle inversion. unaryMinus() should be used instead and this function should be removed.